setup.py

```diff
@@ -107,7 +107,17 @@ def run(self):
         import build_sphinx
         build_sphinx.rebuild()
 
-if 'clean' not in sys.argv:
+# determine if this run requires building of Qt GUI ui->py
+build_qt = True
+# _standard_ commands which shouldn't trigger the Qt build
+no_build_commands = ['clean', 'sdist', 'bdist', 'check', 'upload', 'register', '--help']
+
+for c in no_build_commands:
+    if any([c in b for b in sys.argv]):
+        build_qt = False
+        break
+
+if build_qt:
     _ = subprocess.call([sys.executable, "src/sas/qtgui/convertUI.py"])
```

Comment:
Not sure why there is the spurious `_ = exp` here.

Reply:
> Not sure why there is the spurious `_ = exp` here.

`subprocess.call` returns a `returncode` and I want to be explicit that I'm ignoring it.